dev/dartdoc.dart

```diff
@@ -56,6 +56,22 @@ dependencies:
     args.add(name.substring(0, name.length - 5));
   }
 
+  [
```

Comment:
How would you like this logic to fail if that directory can't be opened? Silently, or with log, or exception?